_config.yml

```diff
@@ -43,10 +43,18 @@ defaults:
       type: events
     values:
       layout: event
+  - scope:
+      path: ""
+      type: resources
+    values:
+      layout: resource
 
 collections:
   events:
     output: true
     permalink: /:year-:month-:day.html
   posts:
     permalink: /:year-:month-:day-:title.html
+  resources:
+    output: true
+    permalink: /:title.html
```

Comment:
These links should work without the `.html` extension, right? I think we can remove the `html` extension from all of these links.

Should we reserve that first level of the permalink for the type of link, for example `resource/:title`?